commands/dns_cli.py

```diff
@@ -23,21 +23,30 @@
 AWS_CERT_REGION = "eu-west-2"
 
 
-def wait_for_certificate_validation(acm_client, certificate_arn, sleep_time=5, timeout=600):
-    click.secho("waiting for cert...", fg="yellow")
+def wait_for_certificate_validation(acm_client, certificate_arn, sleep_time=10, timeout=600):
+    click.secho(f"Waiting up to {timeout} seconds for certificate to be validated...", fg="yellow")
     status = acm_client.describe_certificate(CertificateArn=certificate_arn)["Certificate"]["Status"]
     elapsed_time = 0
     while status == "PENDING_VALIDATION":
-        if elapsed_time > timeout:
-            raise Exception(f"Timeout ({timeout}s) reached for certificate validation")
+        if elapsed_time >= timeout:
+            raise Exception(
+                f"Timeout ({timeout}s) reached for certificate validation, might be worth checking things in the AWS Console"
+            )
         click.echo(
             click.style(f"{certificate_arn}", fg="white", bold=True)
-            + click.style(f": Waiting {sleep_time}s for validation, {elapsed_time}s elapsed...", fg="yellow"),
+            + click.style(
+                f": Waiting {sleep_time}s for validation, {elapsed_time}s elapsed, {timeout - elapsed_time}s until we give up...",
+                fg="yellow",
+            ),
         )
         time.sleep(sleep_time)
         status = acm_client.describe_certificate(CertificateArn=certificate_arn)["Certificate"]["Status"]
         elapsed_time += sleep_time
-    click.secho("cert validated...", fg="green")
+
+    if status == "ISSUED":
+        click.secho("Certificate validated...", fg="green")
+    else:
+        raise Exception(f"""Certificate validation failed with the status "{status}".""")
```

Comment:
It could be something other than ISSUED after PENDING_VALIDATION, I figured it would be helpful to know, and to stop execution at that point.